addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4677,7 +4677,165 @@ registry.DynamicSvg = SnippetOptionWidget.extend({
     },
 });
 
+/**
+* Allows to copy the anchor to clipboard and optionally rename the anchor.
+*/
+const CopyAnchorToClipboardAndNotifyWidget = SnippetOptionWidget.extend({
+
+    /**
+    * @override
+    */
+    start: function () {
+        if (this.$trigger) { // the element that triggers the copy to clipboard and the notification
+            const clipboardParams = {
+                trigger: this.$trigger[0],
+                options: {
+                    text: () => this._getAnchorLink(),
+                }
+            };
+            this._setupClipboardJSAndNotification(clipboardParams, () => {
+                const anchor = decodeURI(clipboardParams.options.text());
+                return {
+                    type: 'success',
+                    message: _.str.sprintf(_t("Anchor copied to clipboard<br>Link: %s"), anchor),
+                    buttons: [{text: _t("Edit"), click: () => this._openAnchorDialog(), primary: true}],
+                };
+            });
+            return this._super.apply(this, arguments);
```

Comment:
- `_super` has to be called even if there is no `$trigger`
- Use the ES6 form for that now: `this._super(...arguments)`